include/pacemaker-internal.h

```diff
@@ -10,6 +10,8 @@
 #ifndef PCMK__PACEMAKER_INTERNAL__H
 #define PCMK__PACEMAKER_INTERNAL__H
 
+#define PCMK__DIRECT_INCLUDE
```

Comment:
I wonder if we should do these per-header. Like `PCMK__DIRECT_INCLUDE_PCMKI` or `PCMK__IN_PACEMAKER_INTERNAL_H` or something. (`PCMK__DIRECT_INCLUDE_PACEMAKER_INTERNAL__H` feels too long but it's even clearer.)

We already have `PCMK__PACEMAKER_INTERNAL__H`, etc., but those remain defined at the end of the header. We need one that we `#undef` at the end, as we're doing here with `PCMK__DIRECT_INCLUDE`.

My thinking is that we don't want any OTHER header that defines `PCMK__DIRECT_INCLUDE` to be able to include these sub-header files.